libs/blocks/tabs/tabs.js

```diff
@@ -101,11 +101,19 @@ function getUniqueId(el, rootElem) {
   return [...tabs].indexOf(el) + 1;
 }
 
+function stopEventBubbling(event) {
+  event.stopPropagation();
+}
+
 function configTabs(config, rootElem) {
   if (config['active-tab']) {
     const id = `#tab-${CSS.escape(config['tab-id'])}-${CSS.escape(getStringKeyName(config['active-tab']))}`;
     const sel = rootElem.querySelector(id);
-    if (sel) sel.click();
+    if (sel) {
+      sel.addEventListener('click', stopEventBubbling, false);
+      sel.click();
+      sel.removeEventListener('click', stopEventBubbling, false);
+    }
```

Comment:
Since we are removing the event right after the click can we use below code to trigger it once. Also for third argument `false` is the default value for event listener so I believe we can avoid passing it.

```suggestion
    if (sel) {
     sel.addEventListener('click', stopEventBubbling, { once: true });
     sel.click();
   }
```

Reply:
Good eye! Done.